README.md

```diff
@@ -0,0 +1,48 @@
+# Django SES Plus
+
+Django module to store and send email with AWS SES. It's an extension for [django-ses](https://github.com/django-ses/django-ses) package.
+
+## Releases
+
+You can see the releases [here](https://github.com/Hipo/django-ses-plus/releases).
+
+## Installation
+
+1. pip install `django-ses-plus`. 
+
+2. Add **django_ses_plus** to the `INSTALLED_APPS` in the settings file.
+
+3. Configure settings.
```

Comment:
Shouldn't we set `EMAIL_BACKEND = 'django_ses.SESBackend'` in `settings.py`?

Reply:
`django_ses_plus.SESBackend` This is a great idea, thank you. I will add.

Reply:
Fixed.